src/benchmark.cc

```diff
@@ -513,6 +517,15 @@ size_t RunSpecifiedBenchmarks(BenchmarkReporter* display_reporter,
   return benchmarks.size();
 }
 
+namespace {
+// stores the time unit benchmarks use by default
+TimeUnit default_time_unit = kNanosecond;
+}  // namespace
+
+TimeUnit GetDefaultTimeUnit() { return default_time_unit; }
+
+void SetDefaultTimeUnit(TimeUnit unit) { default_time_unit = unit; }
```

Comment:
do you need this setter method?  seems like just inlining the assignment below in `SetDefaultTimeUnitFromFlag` is just as readable

Reply:
This function is exposed to the header so you can set the default from your app if you'd like that. I use this function below instead of inlining to have a single point where this global is set.